app/flows/check_uk_visa_flow/outcomes/_electronic_travel_authorisation_advice.erb

```diff
@@ -0,0 +1 @@
+^If you're travelling after 15 November 2023, you'll need to apply for an [electronic travel authorisation (ETA)](/electronic-travel-authorisation) instead of an electronic visa waiver. You'll be able to apply for an ETA from 25 October 2023.
```

Comment:
Looks like the file doesn't have an end line (hence the wee stop sign) - could we please add one in?